### ISSUE TITLE
About the scripts of obtaining the offline dataset adaptec1_small.pkl and some confusions

### ISSUE THREAD

**Issue by tomqingo** [laiyao1/chipformer#4]
@laiyao1 

Dear Lai Yao,

We could start to run the codes and obtain some results in Table 2. We are grateful for your help. However, some points need to be clarified and some implementations differ from those described in the paper. 

1) The adaptec1_small.pkl is given to us. I wonder whether the codes of how to form the adaptec1_small.pkl could also be public to us, allowing us to check whether the calculated mask and the other information are correct.

2) We find that in the training data, the meta data is negative. However, the meta data means the width and the height of the macros. I am confused why the meta data is negative. (That is why I seek if it is possible to get the codes for constructing the training dataset)

3) The paper says the topology information would be encoded with the graph VGAE. However, we cannot find a part about the implementation. The circuit information is simply the combination of the size and degree of the macro. We wonder why there are some differences between the implementations and the papers:-)

I'm looking forward to hearing back from you!

Thank you.

**Comment by laiyao1** [comment]
Dear tomqingo,

Thank you for your questions.

1. As "Offline Data Collection." in our paper, we collect the data based on MaskPlace. You can refer to the code in MaskPlace

2. The meta-data has been under normalization. We do not use it as input in our model as described in papers. Our dataset document has also already clarified that (https://docs.google.com/document/d/1q7dajZt3kIVpyRbvfPSQ7H6stM1Ahz8A2YoyZ2dxJE4/edit?usp=sharing).

3. The VGAE paper is an optional module. We do not include the part in this repo at the present time.
